source/_cookbook/configuration_yaml_by_brusc.markdown

```diff
@@ -8,6 +8,6 @@ comments: false
 sharing: true
 footer: true
 ha_category: Example configuration.yaml
-ha_external_link: https://github.com/brusc/Home-Assistant-Configuration
+ha_external_link: https://github.com/oakbrad/brad-homeassistant-config
```

Comment:
If you want to add your configuration then you need to create a new file rather than editing an existing one.